### PR TITLE
[Preview2] Make dotnet-ef a global tool

### PR DESCRIPTION
**Pull request by bricelam**
Related PRs:
- aspnet/Universe#973
- aspnet/Coherence-Signed#807
- dotnet/cli#8844

Part of #10554

### CODE REVIEW

File: EFCore.sln [hunk 1]
```diff
@@ -29,6 +29,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{CE6B50B2-34A
 	EndProjectSection
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "dotnet-ef", "src\dotnet-ef\dotnet-ef.csproj", "{2D66A1DA-D102-4DD9-960B-7D863BBB53DE}"
+	ProjectSection(ProjectDependencies) = postProject
+		{4F7C93F3-A30F-4061-804C-32293DC256A1} = {4F7C93F3-A30F-4061-804C-32293DC256A1}
+	EndProjectSection
```
Comment (bricelam) [inline EFCore.sln]: @natemcmaster I was getting `dotnet pack` errors when I tried to add `<ProjectReference Include="..\ef\ef.csproj" />` to dotnet-ef.csproj. Have you encountered anything similar?

> error : Could not find a part of the path 'src\ef\bin\Debug\netcoreapp2.0\publish'.

Comment (natemcmaster) [reply EFCore.sln]: Yes, I've seen this. I couldn't find an issue for this already, so I opened this: https://github.com/dotnet/sdk/issues/2047

It's the reason that aspnet/Scaffolding and aspnet/DotNetTools still use a .nuspec file for these tools

**Comment by bricelam** [comment]
Blocked waiting for dotnet/cli#8705

File: NuGetPackageVerifier.json [hunk 2]
```diff
@@ -4,17 +4,34 @@
             "AdxVerificationCompositeRule"
         ],
         "packages": {
+            "dotnet-ef": {
+                "packageTypes": [
+                    "DotnetTool"
+                ],
+                "Exclusions": {
+                    "NEUTRAL_RESOURCES_LANGUAGE": {
+                        "tools/netcoreapp2.0/any/Newtonsoft.Json.dll": "Extenal"
```
Comment (natemcmaster) [inline NuGetPackageVerifier.json]: Typo: Extenal => External

**Pull request merged**